### ISSUE TITLE
Empty grid despite items existing, 0.10.0-beta1

### ISSUE THREAD

**Issue by Puffton** [STRML/react-grid-layout#145]
The grid is always empty in 0.10.0-beta1 (checking the HTML it's just an empty div), I have the following:
```html
<ResponsiveReactGridLayout onLayoutChange={this.layoutChange}
  onBreakpointChange={this.breakpointChange}
  onResize={this.resize}
  isDraggable={this.props.stackType == "template"}
  isResizable={this.props.stackType == "template"}
  {...this.props}>
    {$.map(this.state.items, this.createElement)}
</ResponsiveReactGridLayout>
```

The code is unchanged from before the upgrade (except changes to pave way for React 0.14.x). The list of items (this.state.items) contains multiple elements and in this.createElement it is evident from logging that the map is working.

No errors, no warnings. All I see is:
```html
<div width="1126" cols="40" id="13" class="react-grid-layout page"
  data-reactid=".0.0.4.0.0"
  style="color: rgb(204, 204, 204); border-color: rgb(156, 159, 163); width: 1125.52px;
    height: 796.687px; background-image: none; background-color: rgb(44, 51, 56);">
</div>
```
The difference being that the data-reactid changed from ".0.4.0.0", and that all items disappeared. Requiring it like this:
```js
global.ResponsiveReactGridLayout = require('react-grid-layout').Responsive;
```

Same in all major browsers I have tried (Chrome, Safari, Firefox) on latest OS X-version.

EDIT: Stupid me, 0.10.0-beta1 does *not* have the fix for non-numeric: https://github.com/STRML/react-grid-layout/commit/b4bb88d002ce2c855be20dc212c868711099f1c7, would it be possible to release a 0.10.0-beta2 of the current trunk?

**Comment by fractalawareness** [comment]
The same issue here, nothing is being shown.

**Comment by runeli** [comment]
Same here. Mapping the property i to a number does not solve the issue.

**Comment by Puffton** [comment]
I edited the beta1 source, removing parseInt, then it worked for me.

**Comment by STRML** [comment]
This is fixed in `0.10`+.
